src/Analysis/GWASResultsV2/Views/Results/Results.jsx

```diff
@@ -1,16 +1,150 @@
-import React, { useContext } from 'react';
+import React, { useContext, useState } from 'react';
+import { useQuery } from 'react-query';
+import { Spin, Button } from 'antd';
+
 import DetailPageHeader from '../../SharedComponents/DetailPageHeader/DetailPageHeader';
 import SharedContext from '../../Utils/SharedContext';
+import {
+  fetchPresignedUrlForWorkflowArtifact,
+  queryConfig,
+} from '../../Utils/gwasWorkflowApi';
+import '../../../GWASApp/GWASApp.css';
 
 const Results = () => {
+  const [imageLoaded, setImageLoaded] = useState(false);
+  const [imageLoadFailed, setImageLoadFailed] = useState(false);
   const { selectedRowData } = useContext(SharedContext);
   const { name, uid } = selectedRowData;
+
+  const { data, status } = useQuery(
+    [
+      'fetchPresignedUrlForWorkflowArtifact',
+      name,
+      uid,
+      'manhattan_plot_index',
+    ],
+    () => fetchPresignedUrlForWorkflowArtifact(
+      name,
+      uid,
+      'manhattan_plot_index',
+    ),
+    queryConfig,
+  );
+
+  const displayError = (errorMessage) => (
+    <div>
+        ❌ {errorMessage}
```

Comment:
Comment: I've created a LoadingErrorMessage component as part of https://github.com/uc-cdis/data-portal/pull/1266

We might want to refactor this displayError function to use that.

Reply:
Good idea, please do.